### ISSUE TITLE
No sale de la sesión

### ISSUE THREAD

**Issue by laramaktub** [laramaktub/covid19#53]
Cuando uno ya está logeado y le da al botón de Si quieres cerrar sesion y salir, haz click aqui. vuelve a la pantalla inicial, pero cuando se intenta entrar de nuevo, en realidad sigue logeado con la ultima cuenta que estuviera usando. 

Juraría que esto funcionaba antes.

**Comment by pedrosolisgarcia** [comment]
@laramaktub no puedo reproducir el bug. A mi me funciona sin problema. No te sale en ningun caso?

**Comment by laramaktub** [comment]
No...te sale y te deja por ejemplo  conectarte con un usuario distinto?  Porque a mi hace como que sale, pero al volver a darle a empezar el training, veo que sigo logeada

**Comment by pedrosolisgarcia** [comment]
No, ya se lo que debe pasar (lo que no se es por que). La app de desloguea, de lo contrario no verías la ventana de login. Lo que debe ocurrir es que por alguna razón, al lanzar el request de logoeo de nuevo, debe tener la última sesión en caché, y te autologuea con ella. Tendré que investigar si es un problema de refresco en frontend o de seguridad por parte de backend
